### DISCUSSION TITLE
Welcome to auto_annotate Discussions!

### DISCUSSION THREAD

**Discussion by AlvaroCavalcante** [AlvaroCavalcante/auto_annotate#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and give us some insight about this project!

**Comment by guruanbu** [comment]
give u r savedmodel format

**Reply by AlvaroCavalcante** [reply]
Hey @guruanbu, how are you? Could you please explain your doubt or idea? I didn't understand!
